example/redisfailover/pmem.yaml

```diff
@@ -15,14 +14,17 @@ spec:
       - "--protected-mode"
       - "no"
   redis:
+    securityContext:
+      runAsNonRoot: False
     replicas: 3
-    image: redis_pmem # From https://github.com/pmem/redis
+    image: pmem/redis
```

Comment:
Looks like pmem/redis is not tagged. We should request tagging from upstream maintainers of this container image.